### DISCUSSION TITLE
[2주차][온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#148] [General]
**순서**

~22:05 체크인. 자신의 PR 및 주제 작성

~22:20 다른 PR 참조하여 내 PR 수정하기. 궁금한 점, 모르는 점은 PR에 질문달기

~22:40 입풀기(with 카훗 https://kahoot.it/ )

~23:00 3인 모의면접. 1:1로 진행하며, 남은 1명은 관찰자. 

※ 3인 모의면접이 끝나면 회고(스터디 목표, 스터디 좋았던 점, 건의사항, 체크아웃) 작성하고 자유롭게 해산해주세요~

**Comment by bigsaigon333** [comment]
## 체크인: ~22:05

현재 기분을 10점 만점으로, 그리고 그 이유도 간단히 기재해주세요 :smile:

**Reply by yujo11** [reply]
6점 - 우선 순위가 높은 해야할 일들이 쏟아져서 정신을 못 차리고 있습니다.

**Reply by zereight** [reply]
8점: 오랜만에 동료들 만나서 기분이 좋았습니다

**Reply by ritajeong** [reply]
7점 조금 피곤합니다ㅜ

**Reply by ddongule** [reply]
9점 방금 자다 일어나서 깨운하군요!!!!!!!!!!!!

**Reply by bigsaigon333** [reply]
7점: 아침부터 잠실에 나와서 공부중입니다. 하루가 진짜 빠르네요!!

**Reply by jinsoobyeon** [reply]
5점: 의견이 다를때 커뮤니케이션은 역시 어려운것 같아요....

**Reply by Lobo2u** [reply]
7점! 피곤하네요 ㅜㅠ

**Comment by bigsaigon333** [comment]
## 자신의 PR 및 주제 작성

자신의 PR 링크와 어떤 주제에 대해서 작성하였는지 링크를 달아주세요

**Reply by bigsaigon333** [reply]
|주제|이슈 링크|PR 링크|
|:--:|:--:|:--:|
|arguments 객체란? | #143  |#144 |
|rest parameter란? | #141  |#142 |
|DOM이란? | #52  |#140 |
|이벤트 위임이란? | #81  |#138 |
|이벤트 버블링/캡쳐링이란?| #24  | #137 |

**Reply by ritajeong** [reply]
이슈링크 #43 의 [script,async,defer](https://github.com/ritajeong/fe-interview-basecamp/tree/ritajeong/javascript/script%2Casync%2Cdefer)을 정리했습니다. 
commit이 1주차 PR( #63 )에 이어서 올라갔는데, 다시 fork를 해야하는 건지 잘 모르겠습니다.ㅜ

**Reply by zereight** [reply]
- #135 
- #134 
- #133 
- #129 
- #127 
- #126 
- #125 
- #124 
- #123 
- #122 
- #121

**Reply by jinsoobyeon** [reply]
1. 프로토타입이란? #118
2. CORS란? #120
3. Redux에서 Flux란? #128

**Reply by ddongule** [reply]
- ISSUE: #5 PR: #154 
- ISSUE: #27 PR: #153
- ISSUE: #31 PR: #155

**Reply by Tanney-102** [reply]
- virtualDOM: [PR](https://github.com/fe-backpackers/fe-interview-basecamp/pull/150)
- JSX: [PR](https://github.com/fe-backpackers/fe-interview-basecamp/pull/151)
- 클로저: [PR](https://github.com/fe-backpackers/fe-interview-basecamp/pull/152)

**Comment by bigsaigon333** [comment]
## 입풀기(with 카훗)

링크: https://kahoot.it/
PIN 번호:

**Comment by bigsaigon333** [comment]
## 3인 모의면접

인터뷰어 1명, 인터뷰이 1명, 관찰자 1명으로 구성됩니다.

각각 역할을 번갈아가면서 모든 사람이 각 역할을 2번씩 수행해주세요 :+1: 

(총 3번의 1:1 모의면접이 나오겠죵?)


3인 매칭은 랜덤입니다 :+1:

**Comment by ddongule** [comment]
## 체크아웃

**Reply by ritajeong** [reply]
- 스터디 목표
- 스터디 좋았던 점
- 건의사항
- 기타 등등

**Reply by Lobo2u** [reply]
7점!
오늘도 재미있게 공부하고갑니다!
담에는 열심히 공부해서 답변도가능하게!하겠심다 ;)

**Reply by jinsoobyeon** [reply]
10점: 모의 면접 정말 많이 도움되는거 같아요!! 다음 번에는 풀리퀘스트 글쓰는 것보다 더 깊게 공부해서 면접도 대비 해야겠어요~~

**Reply by Tanney-102** [reply]
- 잘 정리하고 있다고 생각했는데 막상 말해보니 그것도 아니었던 것 같다...
- 예시도 잘 정리해놓는게 중요할 듯
- 충분히 생각하고 말하는 연습을 해야할 것 같다.

**Reply by ritajeong** [reply]
10점 : 지난주보다 더 딥한 모의면접을 한 것 같아서 좋았습니다~~ 건의사항으로는 고정주제를 한개나 두개정도로 정해놓아도 좋을 것 같습니다 카훗에서 나오는 주제 중에 생소하거나 잘 모르는 주제가 나오면 어려웠던 적이 종종 있었어요

**Reply by zereight** [reply]
- 스터디 목표
  - 말끔한 js 지식 설명을 위해
- 스터디 좋았던 점
  - 내 지식을 나의 언어로 잘 풀어내는 연습이 되고 있다.
- 건의사항
  - 1번방 저주받았나봐요. 그루밍님이 1번방에서 마이크가 안되요
- 체크아웃
  - 9점: 생각보다 할만한 질문들이 나와서 정답을 말한것같다! 하지만 좀 장황했다고 생각한다.

**Reply by ddongule** [reply]
10쩜!!! 
같이 말하는거 재밌어요옹 ㅎㅎㅎ
- 동기 비동기 충격..! 제대로 정리가 안되어있다 하하하하하하하하하
- 공부 더 해야겠다..

**Reply by bigsaigon333** [reply]
9점

- 막차시간 때문에 급하게 떠나서 죄송합니다 :sob: 
- 묻고 답할 수 있는 귀중한 시간을 날려서 너무 아쉽습니다 :sweat_drops: 
- 정확히 알아야 좋은 피드백을 줄 수 있는 것 같아요. 저도 잘 모르는 주제에 대해서는 어떻게 질문을 해야할지 모르겠어요 :sob: 주제에 따라서 페어를 매칭하던가 해야될거 같네요
- 너무 글에 완성도를 높히려고 안해야겠습니다. 일단은 불완전하더라도 써본다는 것에 의의를!

**Comment by ddongule** [comment]
## 회고
- 스터디 목표
- 스터디 좋았던 점
- 건의사항
- 기타 등등
